test/parallel/test-fs-rm.js

```diff
@@ -137,6 +146,48 @@ function removeAsync(dir) {
       fs.rmSync(filePath, common.mustNotMutateObjectDeep({ force: true }));
     }
   }));
+
+  // Should delete a valid symlink
+  const linkTarget = path.join(tmpdir.path, 'link-target-async.txt');
+  fs.writeFileSync(linkTarget, '');
+  const validLink = path.join(tmpdir.path, 'valid-link-async');
+  fs.symlinkSync(linkTarget, validLink);
+  fs.rm(validLink, common.mustNotMutateObjectDeep({ recursive: true }), common.mustCall((err) => {
+    try {
+      assert.strictEqual(err, null);
+      assert.strictEqual(fs.existsSync(validLink), false);
+    } finally {
+      fs.rmSync(linkTarget, common.mustNotMutateObjectDeep({ force: true }));
+      fs.rmSync(validLink, common.mustNotMutateObjectDeep({ force: true }));
+    }
+  }));
+
+  // Should delete an invalid symlink
+  const invalidLink = path.join(tmpdir.path, 'invalid-link-async');
+  fs.symlinkSync('definitely-does-not-exist-async', invalidLink);
+  fs.rm(validLink, common.mustNotMutateObjectDeep({ recursive: true }), common.mustCall((err) => {
```

Comment:
I think you meant `invalidLink` here. Isolating each test in a code block could help catch this kind of typos.
```suggestion
  fs.rm(invalidLink, common.mustNotMutateObjectDeep({ recursive: true }), common.mustCall((err) => {
```

Reply:
Yes, thanks for catching it. Fixed.

> Isolating each test in a code block could help catch this kind of typos.

Agree.